src/pal/src/loader/module.cpp

```diff
@@ -220,7 +222,7 @@ LoadLibraryExW(
         return NULL;
     }
     
-    CHAR lpstr[MAX_LONGPATH];
+    CHAR * lpstr = NULL;
```

Comment:
This cannot work, you assing NULL to the pointer, then call the LOADConvertLibraryPathWideStringToMultibyteString which now ignores the parameter and then you end up calling FILEDosToUnixPathA with NULL parameter.
I guess that's the reason why the PAL tests are failing for this commit.